kip-0014/kip-0014.md

```diff
@@ -0,0 +1,102 @@
+---
+KIP: "0014"
+Title: Updated KIP-0012 Account Protocols
+Author: Stuart Popejoy stuart@kadena.io, Doug Beardsley doug@kadena.io
+Status: Draft
+Type: Standard
+Category: Chainweb
+Created: 2021-12-22
+---
+
+## Abstract
+
+Augment KIP-0012 account protocols with two new protocols, Guard Accounts and Rotatable Accounts.
+Modify KIP-0012 Single-Key Accounts to forbid rotation.
+
+## Motivation
+
+KIP-0012 introduced Account Protocols and the Single-Account protocol,
+as implemented in the KDA `coin` contract and elsewhere.
+KIP-0012 reserves all account names starting with `C:` where `C` is a single alpha protocol identifier.
+
+The KIP-0012 Single-Account protocol specifies `k:KEY` where KEY is a hex lowercase ED25519 public key.
+This functions to _reserve the account name_ on any chain which avoids problems like
+"squatting" where an account for a given name on one chain is grabbed by a malicious actor on
+another chain. `k:` has effectively neutralized this threat vector.
+
+However, `k:` is deficient in the following ways:
+
+### Rotation considered harmful for "k:" accounts
+
+Under `fungible-v2` and other standards, `rotate` is specified where an account guard can be
+changed. This presents a challenge to many systems that are accustomed to single-key standards
+on other blockchains such as Bitcoin and Ethereum where an "address" is irrevocably tied
+to a single key. As such this presents problems for "total chain analysis" of a given token
+using standard methods like tracking `fungible-v2.TRANSFER` events, which only provide account
+names and not guards.
+
+While rotation has proven useful with `k:` accounts, this is largely because `k:` makes _account
+reservation_ possible, but by allowing rotate, this robs the protocol from a significant benefit
+which is simplified and robust chain accounting.
```

Comment:
Any updates on KIP14? Asking because I'm trying to implement fraud analytics on Kadena and as mentioned here, theoretically a bad actor could rotate a k:account's key, making tracing harder.